site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+---
+title: "Getting to know App resources better with kctrl"
+slug: kcrl-app-commands-blog
+date: 2022-04-28
+author: Soumik Majumder
+excerpt: "Get better visibility into your app resources with kctrl's new app commands and enhancements to the package commands"
+image: /img/kapp-controller.svg
+tags: ['carvel', 'kapp-controller', 'gitops', 'apps', 'kctrl', 'cli']
+---
+
+Since the previous release of `kctrl`, we have been iterating over how we can help users take a closer look at what App CRs are doing on the cluster. This includes ones authored by users and those that are created as a result of package installations.
+
+As promised, the latest release of `kctrl` introduces a set of commands which allow users to observe and interact with App CRs more conveniently. `kctrl` v0.36.0 also introduces an improved status tailing experience which surfaces relevant information while waiting for resources to reconcile.
+
+# The `app` commands
+These commands aim to help users manage App CRs on the cluster better. Lets dig in and see what they look like!
+
+The user can list apps created on the cluster by running,
+```bash
+$ kctrl app list
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
+
+Available apps in namespace 'default'
+
+Name        Status               Since Deploy  Owner                     Age  
+hello-app   Reconcile succeeded  1m            PackageInstall/hello-app  1m  
+simple-app  Reconcile succeeded  22s           -                         1m  
+
+Succeeded
+```
+`kctrl` makes a distinction between App CRs deployed by themselves and the ones created due to a package installation. We can take a closer look at an app by using the `get` command.
+```bash
+$ kctrl app get -a simple-app
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
+
+Namespace         default  
+Name              hello-app  
+Service Account   hello-app-default-sa  
+Status            Reconcile succeeded  
+Owner References  - packaging.carvel.dev/v1alpha1/PackageInstall/hello-app  
+Conditions        - type: ReconcileSucceeded  
+                    status: "True"  
+                    reason: ""  
+                    message: ""  
+
+Succeeded
+```
+In case the app is erroring out, the command displays relevant errors. For instance, an app which is applying a resource with a misspelt "kind" field looks like:
```

Comment:
nit: In case the app `errors out` 

instead of 

In case the app is `erroring out`

Reply:
Adopted!